content/en/mobile_testing/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Mobile Application Testing and Monitoring
+kind: documentation
+description: "Create intelligent, self-maintaining mobile tests to ensure the most critical parts of your mobile applications are up and running from real devices."
+disable_sidebar: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
+  tag: "Blog"
+  text: "Best practices for creating end-to-end tests"
+- link: "/mobile_testing/mobile_app_tests"
+  tag: "Documentation"
+  text: "Learn how to create Synthetic mobile app tests"
+- link: "/mobile_testing/settings"
+  tag: "Documentation"
+  text: "Learn how to upload your iOS or Android mobile applications"
+- link: "/continuous_testing/"
+  tag: "Documentation"
+  text: "Learn about Continuous Testing & CI/CD"
+cascade:
+  algolia:
+    tags: ['mobile_testing']
+---
+
+Mobile Application Testing allows you to test and monitor key business flows for Android and iOS applications using real devices. 
+
+## Record mobile app tests
+
+{{< img src="mobile_testing/mobile_application_testing_demo.png" alt="Examples of the recording workflow for a Synthetic Mobile Test" style="width:80%;">}}
```

Comment:
Let's replace this with a video of the recorder instead of the screenshot so it matches what we have for Browser Tests under Synthetic Monitoring and let's also copy the "Use the Synthetic Monitoring & Continuous Testing Explorer" section from that page as well (https://docs-staging.datadoghq.com/alai97/add-mobile-app-testing-docs/synthetics/)